README.md

```diff
@@ -15,7 +15,7 @@ If you are an openSUSE Tumbleweed user, it comes with the latest Ruby releases a
 
 ```console
 zypper ref
-zypper in ruby ruby-devel ruby2.5-rubygem-bundler nodejs gcc make libxml2-devel libxslt-devel
+zypper in ruby ruby-devel rubygem(bundler) nodejs gcc make libxml2-devel libxslt-devel
```

Comment:
The parentheses need escaping, or else the shell will eat them.
There are different ways to escape, easiest and best looking is what coolo said: `'rubygem(bundler)'`

Reply:
Updated.